aspnetcore/mvc/advanced/app-parts.md

```diff
@@ -95,10 +92,33 @@ The `GenericController` class:
 
 [!code-csharp[](./app-parts/sample2/AppPartsSample/GenericController.cs)]
 
+For example, requesting a URL of `https://localhost:5001/Sprocket` results in the following response:
+
+```text
+Hello from a generic Sprocket controller.
+```
+
 ### Display available features
 
-The features available to an app can be enumerated by by requesting an `ApplicationPartManager` through [dependency injection](../../fundamentals/dependency-injection.md):
+The features available to an app can be enumerated by requesting an `ApplicationPartManager` through [dependency injection](../../fundamentals/dependency-injection.md):
 
 [!code-csharp[](./app-parts/sample2/AppPartsSample/Controllers/FeaturesController.cs?highlight=16,25-27)]
 
-The [download sample](https://github.com/aspnet/AspNetCore.Docs/tree/master/aspnetcore/mvc/advanced/app-parts/sample2) uses the preceding code to display the app features.
\ No newline at end of file
+The [download sample](https://github.com/aspnet/AspNetCore.Docs/tree/master/aspnetcore/mvc/advanced/app-parts/sample2) uses the preceding code to display the app features:
+
+```text
+Controllers:
+    - FeaturesController
+    - HomeController
+    - HelloController
+    - GenericController`1
+    - GenericController`1
+Tag Helpers:
+    - PrerenderTagHelper
+    - AnchorTagHelper
+    - CacheTagHelper
+    - DistributedCacheTagHelper
+    - EnvironmentTagHelper
+    ...
```

Comment:
There are quite a few Tag Helpers now, so the `...` here denotes that now all of them are shown. I dunno if there's a better convention for that, so let me know if you've done something like this before.